app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardsSettingFragment.kt

```diff
@@ -85,32 +113,20 @@ class CreditCardsSettingFragment : PreferenceFragmentCompat() {
                 .getString(R.string.preferences_credit_cards_sync_cards_across_devices),
             loggedInTitle = requireContext()
                 .getString(R.string.preferences_credit_cards_sync_cards),
-            onSignInToSyncClicked = {
-                val directions =
-                    CreditCardsSettingFragmentDirections.actionCreditCardsSettingFragmentToTurnOnSyncFragment()
-                findNavController().navigateBlockingForAsyncNavGraph(directions)
-            },
-            onReconnectClicked = {
-                val directions =
-                    CreditCardsSettingFragmentDirections.actionGlobalAccountProblemFragment()
-                findNavController().navigateBlockingForAsyncNavGraph(directions)
-            }
+            onSignInToSyncClicked = { navigateToSyncFragment() },
+            onReconnectClicked = { navigateToAccountProblemFragment() }
```

Comment:
I don't think this change is really necessary since the functions are not used more than once. It would be preferable to avoid unrelated changes in such a big PR and file followups for these sort of things. We could also potentially avoid `TooManyFunctions ` by undoing some of this.

Reply:
I disagree, I think it makes this much easier to read

Reply:
I think that thinking is good, but I don't think that is really the issue here. It should be easy enough to read the navigation code as-is since it's only 3 lines and could be 2 if we inlined the `directions`. So, I would be a bit considerate on how we write our commits with consideration on helping streamline the review process.